### DISCUSSION TITLE
System Capabilities/Limits

### DISCUSSION THREAD

**Discussion by ahogan27** [geezacoleman/OpenWeedLocator#35] [General]
Hi @geezacoleman,

It's mentioned in the readme that the current design is setup for 1 OWL unit per boom metre, which can control 4 nozzles. Has there been any testing on the limitations of how wide the camera can accurately detect, or how many nozzles can be run from each OWL unit? At the current camera mounting height of 820mm, is the full scan width of the camera 1m wide? Will it still perform as expected if the mounting position is raised?

Cheers,

**Comment by geezacoleman** [comment]
Thanks for the question @ahogan27! 

As you increase the height of the camera to cover more area on the ground, the size of the pixels on the ground increase (i.e lower on-ground resolution) so you would likely decrease performance of the algorithm. But we haven't done any testing on how high the camera could be to reliably detect weeds, so there's opportunity there if you wanted to test some different heights out!

We've based all the calculations for spray width/nozzle number off 4 nozzles per metre, or one nozzle covering 25cm. If you raise it up above what's specified without adjusting the lane size, those lanes won't correspond to the nozzles correctly, so you'll be turning on the middle nozzle but the weed is actually under the outer nozzles. You could control more nozzles, but you would just need to add more lanes and GPIO pins for relays, then also a larger relay board.
